tar/bsdtar.h

```diff
@@ -35,6 +35,10 @@
 #include <stdint.h>
 #include <stdio.h>
 
+#include "elasticarray.h"
+
+ELASTICARRAY_DECL(STRLIST, tapenames_setup, char *);
```

Comment:
I was sort of thinking that the functions would be prefixed `strlist_` as well...